index.html

```diff
@@ -67,7 +68,219 @@ <h1 class="header-text">
+                </p>
+
+                <section class="built-with">
+
+                    <button id="t" type="submit">
+                        html
+                    </button>
+                    <button id="f" type="submit">
+                        css
+                    </button>
+                    <button id="a" type="submit">
+                        javaScript
+                    </button>
+                </section>
+
+                <section class="prompt">
+                    <button type="submit">See Project</button>
+                </section>
+            </div>
+        </div>
+        <div class="card-3">
+            <img src="./images/Snapshoot-Portfolio.svg" alt="snap">
+            <h1 class="title">
+                Tonic
+            </h1>
+            <div class="sub">
+                <p>CANOPY</p>
+                <ul>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+            </div>
+            <div class="article">
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
 
+                <section class="built-with">
+
+                    <button id="m" type="submit">
+                        html
+                    </button>
+                    <button id="q" type="submit">
+                        css
+                    </button>
+                    <button id="v" type="submit">
+                        javaScript
+                    </button>
+                </section>
+                <section class="prompt">
+                    <button type="submit">See Project</button>
+                </section>
+            </div>
+        </div>
+        <div class="card-4">
+            <img src="./images/post.svg" alt="post">
+            <h1 class="title">
+                Multi-Post
+                Stories
+            </h1>
+            <div class="sub">
+                <p>Multi-Post
+                    Stories</p>
+                <ul>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+            </div>
+
+            <div class="article">
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+
+                <section class="built-with">
+
+                    <button id="l" type="submit">
+                        html
+                    </button>
+                    <button id="o" type="submit">
+                        css
+                    </button>
+                    <button id="r" type="submit">
+                        javaScript
+                    </button>
+                </section>
+
+                <section class="prompt">
+                    <button type="submit">See Project</button>
+                </section>
```

Comment:
- [ ] Please ensure to stick to the design guidelines(margin, padding, fonts, width, height etc)
- [ ] As can be seen below the button doesn't match that of the design and padding and margins are not aligned with the original design

### Your Design
![image](https://user-images.githubusercontent.com/43995830/153582835-1785ab1f-9ae8-4b77-ab38-fc39df080d99.png)


### Original Design
![image](https://user-images.githubusercontent.com/43995830/153583148-57c4576b-5d21-4c89-9b3d-ebcf6aa282f2.png)